esp/src/src-react/components/Pods.tsx

```diff
@@ -47,13 +48,23 @@ export const Pods: React.FunctionComponent<PodsProps> = ({
         { key: "divider_1", itemType: ContextualMenuItemType.Divider, onRender: () => <ShortVerticalDivider /> },
     ], [refreshData]);
 
+    const copyButtons = useCopyButtons(columns, selection, "pods");
+
     return <HolyGrail
         header={<CommandBar items={buttons} farItems={copyButtons} />}
         main={
             <SizeMe monitorHeight>{({ size }) =>
                 <div style={{ width: "100%", height: "100%" }}>
                     <div style={{ position: "absolute", width: "100%", height: `${size.height}px` }}>
-                        <Grid height={`${size.height}px`} />
+                        <FluentGrid
+                            data={pods}
+                            primaryID={"name"}
+                            columns={columns}
+                            height={`${size.height}px`}
+                            setSelection={setSelection}
+                            setTotal={setTotal}
+                            refresh={refreshTable}
```

Comment:
Wasn't sure if "sort" was always necessary now? I just saw you'd commented about it being missing in UserGroups down below. Pods does seem to sort by default without it, currently.

Reply:
It should be optional - but that used to cause a warning with the old DojoGrid